app/frontend/src/components/ContributorForm/ContributorForm.tsx

```diff
@@ -218,60 +199,94 @@ export default function ContributorForm({ processForm }: ContributorFormProps) {
             className={classes.textbox}
           />
           <Controller
-            id="contributeWays"
+            id="contribute"
             control={control}
-            name="contributeWays"
+            name="contribute"
             defaultValue=""
-            render={({ onChange, value }) => (
+            render={({ onChange }) => (
               <FormControl>
-                <FormLabel className={classes.radioLabel}>
-                  {CONTRIBUTE_WAYS_LABEL}
+                <FormLabel component="legend" className={classes.radioLabel}>
+                  {CONTRIBUTE_LABEL}
                 </FormLabel>
-                <FormGroup>
-                  {CONTRIBUTE_WAYS_OPTIONS.map((option) => (
+                <RadioGroup
+                  className={classes.contributeRadio}
+                  name="contribute-radio"
+                  onChange={onChange}
+                >
+                  {CONTRIBUTE_OPTIONS.map((option) => (
                     <FormControlLabel
-                      key={option.name}
-                      value={option.name}
-                      control={
-                        <Checkbox
-                          checked={value.includes(option.name)}
-                          onChange={() =>
-                            toggleCheckbox(option.name, value, onChange)
-                          }
-                          name={option.name}
-                        />
-                      }
-                      label={option.description}
+                      key={option}
+                      value={option}
+                      control={<Radio />}
+                      label={option}
                     />
                   ))}
-                </FormGroup>
-                <FormHelperText error={!!errors?.contributeWays?.message}>
-                  {errors?.contributeWays?.message ?? " "}
-                </FormHelperText>
+                </RadioGroup>
               </FormControl>
             )}
           />
-          <Typography
-            variant="body1"
-            htmlFor="expertise"
-            component="label"
-            className={classes.label}
+          <Collapse
```

Comment:
This makes the background image zoom in when the page body gets larger because it is set to `background-size: cover`. Any ideas?

Reply:
Hmm sound strange... is this CSS rule set by MUI somewhere?

Reply:
No its just for the background image to cover the whole area

Reply:
Fixed it by making background fixed